.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,3 +24,7 @@
 - [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
 - [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)
 - [ ] Inform data insights team due to database changes
+- [ ] Make sure all information from the Trello card is in here
+- [ ] Rebased main
+- [ ] Cleaned commit history
```

Comment:
Do people actually do this anymore? There was a period of time where we used to squash all our commits before merging to help maintain a 'cleaner' history but i'm not sure people have been following that.

Reply:
I personally try to squash commits. I guess it's a bit of an art to know what to squash and what not to if you have a big PR and you try to have multiple commits to make it easier to review.

Reply:
My view is that we shouldn't approve if the commit history is not satisfied or not organised. A suggestion is to open a discussion about things to check on a PR before approving?

Reply:
I'd like to discuss this further - lets keep it in for now 👌